app/services/create_github_repo_service/broadcaster.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class CreateGitHubRepoService
+  class Broadcaster
+    MESSAGES = {
+      repository_creation_complete: "Your GitHub repository was created.",
+      import_ongoing: "Your GitHub repository is importing starter code.",
+      create_repo: "Creating GitHub repository.",
+      importing_starter_code: "Importing starter code."
+    }.freeze
+
+    attr_reader :exercise, :message, :message_type, :repo_url
+
+    def self.call(exercise, message, message_type, repo_url = nil)
```

Comment:
See my earlier comment, we aren't re-using `Broadcaster` instances anywhere in the app, I think we should remove the overhead of initializing a object every time we want to broadcast a message and just use class methods instead.